src/pages/Home/Home.css

```diff
@@ -0,0 +1,13 @@
+.app__cta {
```

Comment:
NIT: I would change the classnames to `home__cta` and `home__header` just to be consistent